packages/element/CHANGELOG.md

```diff
@@ -1,37 +1,41 @@
+## Master
+
+-   Graduated `__experimentalCreateInterpolateElement` function to stable api: `createInterpolateElement` (see [20699](https://github.com/WordPress/gutenberg/pull/20699))
+
 ## 2.10.0 (2019-12-19)
 
 ### New Features
 
-- Added `__experimentalCreateInterpolateElement` function (see [17376](https://github.com/WordPress/gutenberg/pull/17376))
+-   Added `__experimentalCreateInterpolateElement` function (see [17376](https://github.com/WordPress/gutenberg/pull/17376))
```

Comment:
I don't know why all the extra spacing was added to the markdown. It happened on saving the file which leads me to believe there was a markdown linting change or prettier is affecting this maybe?